packages/experiments-realm/crm/activity-type/email.gts

```diff
@@ -0,0 +1,20 @@
+import { FieldDef } from 'https://cardstack.com/base/card-api';
+import { contains, field } from 'https://cardstack.com/base/card-api';
+import { StringField } from 'https://cardstack.com/base/card-api';
+import { EmailField } from '../../email';
+import DatetimeField from 'https://cardstack.com/base/datetime';
+
+export class Email extends FieldDef {
+  static displayName = 'Email Activity';
+  @field emailAddress = contains(EmailField);
+  @field emailContent = contains(StringField);
+  @field sentTime = contains(DatetimeField);
```

Comment:
Wouldn't sentTime be the timestamp? If it's meant to be for scheduling, maybe a clearer field name would be better. Is this the actual content of the email? If it is necessary, maybe it should be a markdown field.